### PR TITLE
Review strict-stm cabal file

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: strict-stm/strict-stm.cabal [hunk 1]
```diff
@@ -1,7 +1,9 @@
 name:                strict-stm
 version:             0.1.0.0
-synopsis:            Type classes for concurrency with STM, ST and timing
--- description:
+synopsis:            Strict STM interface polymorphic over stm implementation.
+description:         The `strict-stm` package give a strict interface to stm,
```
Comment (dnadales) [inline strict-stm/strict-stm.cabal:4]:
```suggestion
description:         The `strict-stm` package gives a strict interface to stm,
```

**Review by dnadales — APPROVED** [review]
LGTM

File: strict-stm/strict-stm.cabal (hunk 1 above)
Comment (coot) [reply strict-stm/strict-stm.cabal]: Thanks, included.

**Comment by coot** [comment]
bors merge

**Comment by coot** [comment]
bors merge

**Pull request merged**